features/admin.applications.v1/constants/application-management.ts

```diff
@@ -41,6 +41,11 @@ export class ApplicationManagementConstants {
     public static readonly SYSTEM_APPS: string[] = [ this.CONSOLE_APP_NAME ];
     public static readonly DEFAULT_APPS: string[] = [ this.MY_ACCOUNT_APP_NAME ];
 
+    /**
+     * Application latest version.
+     */
+    public static readonly LATEST_VERSION: string = "v1.0.0";
```

Comment:
Is hardcoding this here intentional?

Reply:
Let's add a comment

Reply:
0bd8ceec802ce087b6d7e10d520ad8d787c6e5f1